LOAN/AllLOAN.rdf

```diff
@@ -56,10 +56,12 @@
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/LOAN/AllLOAN/">
 		<rdfs:label>Loans (LOAN) Domain</rdfs:label>
 		<dct:contributor>Adaptive, Inc.</dct:contributor>
+		<dct:contributor>FIUTUR</dct:contributor>
```

Comment:
Shouldn't this read 'Fiùtur'?

Reply:
@merelog I had that everywhere, but the hygiene tests errored off due to the special character.